compose/material/material/build.gradle

```diff
@@ -21,7 +21,7 @@ plugins {
     id("AndroidXPlugin")
     id("com.android.library")
     id("AndroidXComposePlugin")
-    id("AndroidXPaparazziPlugin")
+    //id("AndroidXPaparazziPlugin") // isn't supported on Windows
```

Comment:
Where it's used? it's safe to just remove it from common build script?

Reply:
It is used only `src/test`, which shouldn't be run in multiplatform-mode (AOSP doesn't use this mode). Tests for common usually go to `commonTest`